package.json

```diff
@@ -71,22 +72,30 @@
       "@semantic-release/commit-analyzer",
       "@semantic-release/release-notes-generator",
       "@semantic-release/github",
+      [
+        "semantic-release-plugin-update-version-in-files",
+        {
+          "files": [
+            "payload-examples/package.json"
+          ]
+        }
+      ],
```

Comment:
@wolfy1339 can you recall why we added this? I don't think we have to update the version file in `payload-examples/package.json` manually, it will be replaced by the `npm` plugin right before the release?

Reply:
It was to update the version of the dependency on `@octokit/webhooks-types`

That way the version would not be out of sync

Reply:
got it thanks!